src/platform/lumin-runtime/elements/builders/image-builder.js

```diff
@@ -161,9 +161,10 @@ export class ImageBuilder extends UiNodeBuilder {
     return spinner;
   }
 
-  _removeMaskAndSpinner (element, prism, spinner) {
+  _removeMaskAndSpinner (element, prism, spinner, properties) {
     // Remove color mask
-    this._callNodeAction(element, 'setColor', [1, 1, 1, 1]);
+    const color = this.getPropertyValue('color', [1, 1, 1, 1], properties);
```

Comment:
I think you should convert the color value to [r,g,b,a] format here. There is exception if I provide color as "red" and the spinner never disappears.